hieradata/dev/wso2/wso2is/pattern-5/default.yaml

```diff
@@ -0,0 +1,512 @@
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+classes:
+  - wso2is
+
+wso2::service_name: wso2is
+wso2::hostname: is.dev.wso2.org
+wso2::mgt_hostname: is.dev.wso2.org
+
+wso2::enable_thrift_service: false
+
+wso2::template_list:
+#  - repository/conf/datasources/bps-datasources.xml
+#  - repository/conf/datasources/metrics-datasources.xml
+  - repository/conf/identity/identity.xml
+#  - repository/conf/identity/entitlement.properties
+#  - repository/conf/identity/CaptchaConfig.properties
+#  - repository/conf/identity/identity-event.properties
+#  - repository/conf/identity/identity-mgt.properties
+#  - repository/conf/identity/sso-idp-config.xml
+#  - repository/conf/identity/application-authentication.xml
+#  - repository/conf/identity/EndpointConfig.properties
+#  - repository/conf/email/email-admin-config.xml
+#  - repository/conf/datasources/am-datasources.xml
+  - repository/conf/axis2/axis2.xml
+#  - repository/conf/security/cipher-text.properties
+#  - repository/conf/security/cipher-tool.properties
+#  - bin/ciphertool.sh
```

Comment:
If this module is being based on latest `wso2base` then these should be in the file list, instead of the template list.

Reply:
Based on wso2base 1.0.0